templates/vue/src/components/modals/NodeModal/forms/ContentForm/VideoForm.vue

```diff
@@ -1,16 +1,52 @@
 <template>
-  <div>
-    <b-form-group label="Video URL">
-      <file-upload
-        id="node-video-media-url"
-        v-model="node.typeData.mediaURL"
-        input-test-id="node-video-url"
-        placeholder="Enter URL for MP4 or YouTube video"
-        required
-        @isUploading="handleUploadChange"
-      />
-    </b-form-group>
-  </div>
+  <b-container>
```

Comment:
When you use `b-container`, bootstrap adds some additional margins which are not necessary here. Please avoid using this unless you want those margins. You can just use a div.
```suggestion
  <div>
```

Reply:
The layout has been fixed, it just uses a div now.